controllers/disruption_controller.go

```diff
@@ -74,6 +74,7 @@ type DisruptionReconciler struct {
 	log                                   *zap.SugaredLogger
 	InjectorServiceAccountNamespace       string
 	InjectorNetworkDisruptionAllowedHosts []string
+	ExpiredDisruptionGCDelay              int
```

Comment:
Same question as for the duration field, why not using a `time.Duration` here? If we keep an `int`, could we eventually specify the unit in the same as you did for the duration field? Like `ExpiredDisruptionGCSeconds` or something like this.